tasks/Debian.yml

```diff
@@ -2,7 +2,7 @@
 - name: Install packages
   package:
     name: "{{ pkg }}"
-    state: present
+    state: "{{ 'latest' if openio_package_upgrade | d(false) else 'present' }}"
```

Comment:
` d(false)` can you define it `default/main.yml`

Reply:
:+1: & LGTM otherwise